packages/ipfs-core/src/components/dht.js

```diff
@@ -69,7 +69,7 @@ module.exports = ({ libp2p, repo }) => {
      * @param {number} [options.numProviders] - maximum number of providers to find
      * @returns {AsyncIterable<{ id: CID, addrs: Multiaddr[] }>}
      */
-    findProvs: withTimeoutOption(async function * (key, options) { // eslint-disable-line require-await
+    vs: withTimeoutOption(async function * (key, options) { // eslint-disable-line require-await
```

Comment:
Why did this method name change from `findProvs` to `vs`?

Reply:
```suggestion
    findProvs: withTimeoutOption(async function * (key, options) { // eslint-disable-line require-await
```

Reply:
Thanks for catching this. Not sure how this happened.